package.json

```diff
@@ -10,7 +10,7 @@
     "format": "prettier --single-quote --trailing-comma=es5 --write src/*.js src/**/*.js src/**/*.jsx",
     "lint:style": "stylelint \"src/**/*.css\"",
     "lint:style:ci": "stylelint \"src/**/*.css\"",
-    "check": "yarn run test && yarn run flow && yarn run lint && yarn run lint:style",
+    "check": "yarn run format && yarn run test && yarn run flow && yarn run lint && yarn run lint:style",
```

Comment:
Hey @jackyzhen this broke CI https://travis-ci.org/apicollective/ui/builds/260876329 as prettier isn't installed. Not sure if you can install prettier as a dev dependency and use that in the format command? Want to give it a try?